### DISCUSSION TITLE
Unable to set color, brightness and color_behaviour for HmIP-BSL

### DISCUSSION THREAD

**Discussion by moto0815** [SukramJ/aiohomematic#1809] [Q&A]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic) right now
- [X] I have read the [FAQs](https://github.com/danielperna84/custom_homematic#frequently-asked-questions) right now
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] I am aware that an integration that was installed via HACS also needs to be updated via HACS
- [X] The Backend (CCU/Homegear/...) is working as expected, devices are taught in and are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

There is no action available for device HmIP that allows to set values for color, brightness and color_behaviour.

### What version of HomematicIP (local) has the issue?

2.0.1

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

CCU3

### Which version of your homematic backend are you running?

3.75.7

### What hardware are you running for your system?

Home Assistant Green

### Which config details do you use

- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [ ] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

### Diagnostics information (no logs here)

It is not about a failure. Instead it is about missing channels

### Additional information

Sensors show correct color and brightness (ch7 and ch11)
Control elemtens (vch 5, 6, 9, 10, 13 and 14) only allow switching on/off or increasing/decreasing value.
Setting values would be required

**Comment by SukramJ** [comment]
Homematc(IP) local 2.0.1 gibt es nicht.
CCU Version 3.75.7 ist veraltet.
Aktuelle HmIP-BSL Firmware ist 2.0.2.

Wie versuchst Du Farbe und Helligkeit bei deinem BSL zu ändern? Screenshot sind hilfreich.

**Reply by SukramJ** [reply]
Da ich mit meinem BSL keine Probleme feststellen konnte habe ich mal eine Diskussion draus gemacht.

**Comment by moto0815** [comment]
Sorry Version Homematic(IP) local müsste 1.68.1 sein.

Ich möchte die Farbe über eine Automation ändern.

Nach Auswahl des Gerätes kann ich allerdings nur Aktionen auswählen, bei den Kanäle ein-/aus-/umgeschaltet werde können bzw. Werte "verringert" oder "erhöht" werden können. Eine feste Farbe (RGB) oder ein Helligkeitswert können nicht eingetragen werden.

![Screenshot](https://github.com/user-attachments/assets/11cf00a8-5c52-42b5-9157-5a5cb1ce886d)

Ein YAML Code Beispiel wäre auch hilfreich.

> Homematc(IP) local 2.0.1 gibt es nicht. CCU Version 3.75.7 ist veraltet. Aktuelle HmIP-BSL Firmware ist 2.0.2.
> 
> Wie versuchst Du Farbe und Helligkeit bei deinem BSL zu ändern? Screenshot sind hilfreich.

**Reply by SukramJ** [reply]
> Ein YAML Code Beispiel wäre auch hilfreich.

Allgeineiner HA Support ist hier nicht das Thema.

Unter Action statt Gerät Leuchte auswählen, dann kann man die Farbe …… ändern.

**Reply by moto0815** [reply]
Super, danke!! Problem gelöst.
